### PR TITLE
feat: add how-to-guide on reading Google Spreadsheets

### PR DESCRIPTION
**Pull request by shrutimantri**

### CODE REVIEW

File: content/docs/15.how-to-guides/google-spreadsheets.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+
+Learn step-by-step how to read data from Google Spreadsheet in a Kestra flow.
+
+You can use any Google Spreadsheet for this tutorial. In case you do not have Google Spreadsheet, you can:
+
+1. Download the data from [this link](https://huggingface.co/datasets/kestra/datasets/raw/main/csv/orders.csv) and save it locally as `orders.csv` file.
+2. Create a new Google Spreadsheet.
+3. Navigate to the `File` menu on the top, and select `Import` option.
+4. Navigate to the `Upload` tab, and click on the `Browse` button.
+5. Select the recently created `orders.csv` file, and click on `Open` button at the buttom of the popup.
```
Comment (wrussell1999) [inline content/docs/15.how-to-guides/google-spreadsheets.md:14]:
```suggestion
5. Select the recently created `orders.csv` file, and click on `Open` button at the bottom of the popup.
```

File: content/docs/15.how-to-guides/google-spreadsheets.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+
+![sheet_import](/docs/how-to-guides/google-spreadsheets/sheet_import.png)
+
+![sheet_upload](/docs/how-to-guides/google-spreadsheets/sheet_upload.png)
+
+![sheet_import_data](/docs/how-to-guides/google-spreadsheets/sheet_import_data.png)
+
+![sheet_uploaded_data](/docs/how-to-guides/google-spreadsheets/sheet_uploaded_data.png)
+
+Now that we have the spreadsheet ready, let us proceed to assign appropriate authorization for the spreadsheet. For this,
```
Comment (wrussell1999) [inline content/docs/15.how-to-guides/google-spreadsheets.md:27]:
```suggestion
Now that we have the spreadsheet ready, let us proceed to assign appropriate authorization for the spreadsheet. For this:
```

File: content/docs/15.how-to-guides/google-spreadsheets.md [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+1. Go to the GCP console, navigate to the [IAM service](https://console.cloud.google.com/iam-admin/iam).
+2. Select `Service accounts` from the left navigation menu.
+3. Click on `Create Service Account` on the top. You may choose to use an existing service account in which case you can skip the next step.
+4. Put in appropriate service account name, service account id (the auto-populated value should be good to start with) and service account description, and click on `Done`.
+
+![create_service_account_1](/docs/how-to-guides/google-spreadsheets/create_service_account_1.png)
+
+![create_service_account_2](/docs/how-to-guides/google-spreadsheets/create_service_account_2.png)
+
+The new service account has been created. Lets add a key to the service account.
```
Comment (wrussell1999) [inline content/docs/15.how-to-guides/google-spreadsheets.md:38]:
```suggestion
The new service account has been created. Let's add a key to the service account.
```

File: content/docs/15.how-to-guides/google-spreadsheets.md [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+
+1. Click on the corresponding service account from the Service Accounts page.
+2. Navigate to `Keys` tab, and click on `Add Key` -> `Create new key`.
+3. On the `Create private key` popup, select `JSON` option, and click on `Create`.
+4. This will download the service account JSON file on your local computer.
```
Comment (wrussell1999) [inline content/docs/15.how-to-guides/google-spreadsheets.md:43]: Add a link to the service account guide we published recently too

File: content/docs/15.how-to-guides/google-spreadsheets.md [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+
+We will now provide access to the spreadsheet for the service account.
+
+1. Copy the email corresponding to the service account from the Service Accounts page.
+2. Go to the spreadsheet, and click on the `Share` button on the top right.
+3. Add the service account email in the `Add people, groups, and calendar events` text box.
+4. You can give the `Viewer` access to the service account.
+5. Click on `Done`.
+
+Let us now enable Google Sheets API in the GCP console.
```
Comment (wrussell1999) [inline content/docs/15.how-to-guides/google-spreadsheets.md:58]:
```suggestion
Let us now enable the Google Sheets API in the GCP console.
```

File: content/docs/15.how-to-guides/google-spreadsheets.md [hunk 6]
```diff
@@ -0,0 +1,102 @@
...
+
+1. On the GCP console, search for `Google Sheets API` service, or directly navigate to one using [this link](https://console.cloud.google.com/marketplace/product/google/sheets.googleapis.com).
+2. Check whether the Google Sheets API is already enabled. If not, you will see an `Enable` button on the page. Click on the `Enable` button.
+
+![enable_google_sheets_api](/docs/how-to-guides/google-spreadsheets/enable_google_sheets_api.png)
+
+With this, we are all set to access the Google Spreadsheet from Kestra flow. Here is how the Kestra flow would look like:
```
Comment (wrussell1999) [inline content/docs/15.how-to-guides/google-spreadsheets.md:65]:
```suggestion
With this, we are all set to access the Google Sheet from Kestra flow. Here is how the Kestra flow would look like:
```

Comment (wrussell1999) [inline content/docs/15.how-to-guides/google-spreadsheets.md:65]:
```suggestion
With this, we are all set to access the Google Spreadsheet from Kestra flow. Here is an example of how the Kestra flow might look like:
```

File: content/docs/15.how-to-guides/google-spreadsheets.md (hunk 4 above)
Comment (shrutimantri) [reply content/docs/15.how-to-guides/google-spreadsheets.md:43]: Done

File: content/docs/15.how-to-guides/google-spreadsheets.md [hunk 7]
```diff
@@ -0,0 +1,104 @@
...
+5. Click on `Done`.
+
+Let us now enable the Google Sheets API in the GCP console.
+
+1. On the GCP console, search for `Google Sheets API` service, or directly navigate to one using [this link](https://console.cloud.google.com/marketplace/product/google/sheets.googleapis.com).
+2. Check whether the Google Sheets API is already enabled. If not, you will see an `Enable` button on the page. Click on the `Enable` button.
+
+![enable_google_sheets_api](/docs/how-to-guides/google-spreadsheets/enable_google_sheets_api.png)
+
+With this, we are all set to access the Google Sheet from Kestra flow. Here is how the Kestra flow would look like:
```
Comment (shrutimantri) [inline content/docs/15.how-to-guides/google-spreadsheets.md:67]:
```suggestion
With this, we are all set to access the Google Spreadsheet from Kestra flow. Here is an example of how the Kestra flow might look like:
```

**Review by wrussell1999 — APPROVED** [review]

**Pull request merged**